modules/apps/notification/notification-service/service.xml

```diff
@@ -34,6 +34,7 @@
 		<column name="subject" type="String" />
 		<column name="to" type="String" />
 		<column name="toName" type="String" />
+		<column name="status" type="int" />
```

Comment:
Can you order it alphabetically including the finder?

Reply:
I tried to order, but the SF didn't let me do it.

Reply:
Got it. Thanks for letting me know. 😉